### DISCUSSION TITLE
CefSharp.Dom.TargetClosedException: 'Protocol error(Runtime.evaluate): Target closed. (Browser Disposed)'

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [cefsharp/CefSharp.Dom#47] [General]
It is extremely annoying

Why I am getting this error ? 

I think the page continuously gets updated with newer data refresh could that be reason? 

Can I freeze state of the page to prevent this error?

`CefSharp.Dom.TargetClosedException: 'Protocol error(Runtime.evaluate): Target closed. (Browser Disposed)'`

![image](https://user-images.githubusercontent.com/19240467/206878259-a18e3912-968e-4c38-a146-24e4aacea9da.png)

**Comment by amaitland** [comment]
> Why I am getting this error ?

How am I supposed to know that from a screenshot of an error message exactly? You need to provide an example of your code. See https://stackoverflow.com/help/minimal-reproducible-example for guidance.
